### ISSUE TITLE
Adding CDN player.js to a page which already has require.js on it will fail

### ISSUE THREAD

**Issue by JodiWarren** [vimeo/player.js#79]
### Expected Behavior
player.js is instantiated without any problems and videos play happily.

### Actual Behavior
The `define` functions clash with the already instantiated RequireJS.

### Steps to Reproduce

1. Load a copy of RequireJS. Having it actual resolve dependencies is unnecessary for issue reproduction.
2. Later, include the CDN link to player.js in a script tag. This is done by a number of external libraries.
3. The summoned video will not play.

I have a quickly knocked-together pen: http://codepen.io/jodi/pen/qqbXxw

Apologies for all the script tags, it's just for easy toggling so you can see the effects each setting/script has.

My suggestion is to provide either a separate `player.iife.js` or something for including via a script tag. Alternatively, maybe provide the iife version by default. Maybe I don't have enough knowledge, but I don't think the user would need the AMD or CommonJS module syntaxes when accessing it via a CDN.

**Comment by bdougherty** [comment]
I've never used RequireJS, but I think you want to load the player library dynamically through RequireJS. I think it's this part of the docs: http://www.requirejs.org/docs/api.html#jsfiles

**Comment by JodiWarren** [comment]
@bdougherty You're absolutely right, that's what you **should** do. However, there are quite a few scenarios where libraries etc are linking to the CND library as a kind of misunderstood best practice. This will cause issues on sites which use RequireJS. For example, all Magento 2 sites make heavy use of RequireJS, and for better or worse, there are lots of people using the library improperly. My hope was that there could be a zero-friction way of increasing compatibility.

Currently you **have** to use the dependency system, you're not able to include it on the page in an old-fashioned script tag.

**Comment by bdougherty** [comment]
@JodiWarren are there any other libraries that work that way? In my experience, most (if not all) use UMD.

**Comment by JodiWarren** [comment]
@bdougherty Not to my knowledge. UMD makes sense in most circumstances. I'm sure there's a better solution than what I've proposed, but I couldn't find one. 

I've now had to fix a few Magento 2 sites which had included player.js via the CDN, so I'm assuming there are more of them out there too. My method has been to generate a custom build of player.js with the Rollup format option set to `iife` and use that instead.

Perhaps just a note in the documentation suggesting doing the above would suffice!
